qtconsole/tests/test_jupyter_widget.py

```diff
@@ -72,11 +74,25 @@ def test_other_output(self):
         ))
 
         # Check proper syntax highlighting
-        if QT6:
+        if QT6 and parse(QT_VERSION) < parse('6.3'):
```

Comment:
Please change this so that the Qt >= 6.3 case comes first and then the other one. I think that makes this test easier to read,